### ISSUE TITLE
Text node contents should not be indented

### ISSUE THREAD

**Issue by poveden** [michaelkourlas/node-xmlcreate#7]
According to the [XML Specification, Section 2.10](https://www.w3.org/TR/xml/#sec-white-space), all characters that are not markup _must_ always be passed on (i.e. significant white space). However, when `pretty` is set to `true` (the default), text nodes are indented, thus altering text nodes' contents.

Actual behavior:
```xml
<?xml version='1.0' encoding='utf-8'?>
<Example>
    <Text>First line
    Second Line
    Third line</Text>
</Example>
```

Expected behavior:
```xml
<?xml version='1.0' encoding='utf-8'?>
index.js:19
<Example>
    <Text>First line
Second Line
Third line</Text>
</Example>
```

**Comment by michaelkourlas** [comment]
This is a known issue which will be fixed in the next release.

**Comment by michaelkourlas** [comment]
Fixed in 2.0.0. A release of js2xmlparser using 2.0.0 will be out shortly.
